src/components/screens/management-screen.tsx

```diff
@@ -0,0 +1,438 @@
+      { name: WorkTimeCategory.INTERNAL, total: timeEntryTotal.internalTime }
+    ];
+
+    const COLORS = [ theme.palette.success.main, theme.palette.warning.main ];
+
+    return (
+      <Paper className={ classes.redirectPersonDetailPaper }>
+        <Box mb={ 2 } width="100%">
+          <UserInfo
+            person={ selectedPersonWithTotalTime.person }
+          />
+        </Box>
+        <Divider/>
+        <Box width="100%" my={ 2 }>
+          { renderExpectedWorkRow(`${strings.startDate}:`, moment(person.startDate).format("DD.MM.YYYY")) }
+          { renderExpectedWorkRow(`${strings.monday}:`, TimeUtils.convertToMinutesAndHours(person.monday)) }
+          { renderExpectedWorkRow(`${strings.tuesday}:`, TimeUtils.convertToMinutesAndHours(person.tuesday)) }
+          { renderExpectedWorkRow(`${strings.wednesday}:`, TimeUtils.convertToMinutesAndHours(person.wednesday)) }
+          { renderExpectedWorkRow(`${strings.thursday}:`, TimeUtils.convertToMinutesAndHours(person.thursday)) }
+          { renderExpectedWorkRow(`${strings.friday}:`, TimeUtils.convertToMinutesAndHours(person.friday)) }
+          { renderExpectedWorkRow(`${strings.saturday}:`, TimeUtils.convertToMinutesAndHours(person.saturday)) }
+          { renderExpectedWorkRow(`${strings.sunday}:`, TimeUtils.convertToMinutesAndHours(person.sunday)) }
+        </Box>
+        <ResponsiveContainer className={ classes.pieChartContainer }>
+          <PieChart>
+            <Pie
+              cx="50%"
+              cy="50%"
+              dataKey="total"
+              data={ workTimeData }
+              label={ renderCustomizedLabel }
+            >
+              { workTimeData.map((entry, index) => (
+                <Cell fill={ COLORS[index % COLORS.length] }/>
+              )) }
+            </Pie>
+            <RechartTooltip content={ renderCustomizedTooltip }/>
+          </PieChart>
+        </ResponsiveContainer>
+        <Box className={ classes.personRedirect }>
+          <Divider/>
+          <Button
+            onClick={ () => handlePersonRedirectClick(person) }
+            className={ classes.personRedirectButton }
+          >
+            <Typography style={{ fontWeight: 600 }}>
+              { strings.managementScreen.seeMore }
+            </Typography>
+            <KeyboardArrowRightIcon/>
+          </Button>
+        </Box>
+      </Paper>
+    );
+  };
+
+  /**
+   * Renders person entry
+   * 
+   * @param personTotalTime person total time data
+   */
+  const renderPersonEntry = (personTotalTime: PersonWithTotalTime) => {
+    const { person, timeEntryTotal } = personTotalTime;
+
+    if (!person.active || !timeEntryTotal || timeEntryTotal.expected === 0) {
+      return;
+    }
+
+    return (
+      <ListItem
+        button
+        disableRipple
+        disableTouchRipple
+        selected={ selectedPersonWithTotalTime?.person.id === person.id }
+        onClick={ event => handleListItemClick(event, personTotalTime) }
```

Comment:
Check comments related to handleListItemClick function itself
```suggestion
        onClick={ () => handleListItemClick(personTotalTime) }
```